src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Views/EmployerRequest/SelectProviderEmail.cshtml

```diff
@@ -48,7 +48,7 @@
                             @foreach (var email in Model.EmailAddresses)
                             {
                                 <div class="govuk-radios__item">
-                                    <input class="govuk-radios__input" type="radio" asp-for="SelectedEmail" value="@email" >
+                                    <input class="govuk-radios__input" type="radio" asp-for="SelectedEmail" id="@email" value="@email" >
```

Comment:
It has not been reported as a bug yet - but this needs to use the id of the model property for the first radio so that the error message can jump to it - like has been done for the SelectedRequests